src/librustc/mir/mod.rs

```diff
@@ -2994,10 +2994,29 @@ pub struct UnsafetyCheckResult {
     pub unsafe_blocks: Lrc<[(hir::HirId, bool)]>,
 }
 
+newtype_index! {
+    pub struct GeneratorField {
```

Comment:
Tempted to call this something other than "field", like "savee", but that sounds really bad.

Reply:
`GeneratorSavedLocal`? =/

Reply:
Oh, I actually like that!

Reply:
Okay, done!